shell/platform/android/io/flutter/plugin/platform/PlatformViewWrapper.java

```diff
@@ -175,12 +177,20 @@ public boolean onInterceptTouchEvent(@NonNull MotionEvent event) {
     return true;
   }
 
+  /** Used on Android O+, {@link invalidateChildInParent} used for previous versions. */
+  @SuppressLint("NewApi")
```

Comment:
why is this required? Do you set a lower targetSdkVersion?

Reply:
I think we should also consider the API levels lower than 26. :)

Reply:
Shouldn't this be a TargetApi attribute instead of suppress lint?

Reply:
Yes, but the right way to express this to the linter is via `@TargetApi` and/or `@RequiresApi` - see https://stackoverflow.com/questions/40007365/requiresapi-vs-targetapi-android-annotations/45773141. SuppressLint works but it looks like we're ignoring a lint without explaining why. It's not a huge deal either way in this case, especially given the comment - but with those lints you don't need the comments since it's encoded in the attribute :)